.circleci/config.yml

```diff
@@ -99,6 +104,16 @@ jobs:
             git config user.name "${CIRCLE_USERNAME}"
             git tag -a v"${version}" -m "version ${version}"
             git push --tags
+  push_pypi:
+    executor: docker-executor
+    steps:
+      - checkout
+      - add_ssh_keys:
+          fingerprints:
+            - '18:3e:7f:b6:39:58:d3:0f:70:54:32:d6:3e:d7:13:4e'
```

Comment:
I feel like the pypi step doesn't need a key. This key is linked to this Github repo.

Though maybe it's safer to leave this because testing if I'm right or not is not easy.

Reply:
This was the only part I didn't understand, which is why I left as-is. I can spend some time later today looking into it.

Reply:
I'm leaving this as-is. I do not understand the implications of removing it.